docker-compose.yml

```diff
@@ -122,6 +122,43 @@ services:
     labels:
       logs: "promtail"
 
+  hyperswitch-web:
+    ports:
+      - "9050:9050"
+      - "9060:9060"
+      - "5252:5252"
+    build:
+      context: ./docker
+      dockerfile: web.Dockerfile
+    networks:
+      - router_net
+    env_file:
+      - ./.env
+    environment:
+      - HYPERSWITCH_PUBLISHABLE_KEY=$HYPERSWITCH_PUBLISHABLE_KEY
+      - HYPERSWITCH_SECRET_KEY=$HYPERSWITCH_SECRET_KEY
+      - HYPERSWITCH_SERVER_URL=${HYPERSWITCH_SERVER_URL:-http://hyperswitch-server:8080}
+      - HYPERSWITCH_CLIENT_URL=${HYPERSWITCH_CLIENT_URL:-http://hyperswitch-web:9050}
+      - SELF_SERVER_URL=${SELF_SERVER_URL:-http://hyperswitch-web:5252}
+    depends_on:
+      hyperswitch-server:
+        condition: service_started
+
+  hyperswitch-control-center:
+    image: juspaydotin/hyperswitch-control-center
+    ports:
+      - "9000:9000"
+    networks:
+      - router_net
+    environment:
+      - apiBaseUrl=http://hyperswitch-server:8080
+      - sdkBaseUrl=http://hyperswitch-web:9050
```

Comment:
The sdkBaseUrl should point to the js assets, in this case it would be `sdkBaseUrl=http://<SDK_SERVICE>/HyperLoader.js`